rocket_sqlite_oracle/src/db/kraken.rs

```diff
@@ -255,9 +255,23 @@ mod wire {
         BitcoinParseAmount(#[from] ParseAmountError),
     }
 
+    #[derive(Debug, PartialEq, Clone, Copy)]
+    pub enum Units {
+        Millicents,
+    }
+
+    impl Units {
+        pub fn units_as_str(&self) -> String {
+            match *self {
+                Units::Millicents => String::from("millicents"),
+            }
+        }
```

Comment:
Three comments in regards to idiomatic Rust code on this snippet:

1. I would omit the `units_` part. Your struct is already called `Units` and you are taking `&self`. No need to repeat yourself here.
2. Functions that are named `as_XYZ` are meant to be cheap conversions without allocations. See https://rust-lang.github.io/api-guidelines/naming.html for reference. In this case, returning `&'static str` would be more appropriate for a function named `as_str`.
3. If you wanted a function to return `String`, a better way would be to implement `std::fmt::Display`. Every type that implements `Display`, automatically implements `ToString`. See https://doc.rust-lang.org/std/string/trait.ToString.html.